e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -100,6 +103,8 @@ function autoscript() {
       mergeMap(
         (tokens) => {
           console.log('Force renew OAuth tokens');
+          // Pause for 2 seconds to allow expired tokens to expire
+          rxDelay(tokenExpiredDelay);
```

Comment:
I'm assuming this works, but I would lose the bet if I was asked if it did. Being that the `rxDelay` is not "attached" to any event "stream", I would think the JS engine would evaluate it, but then just move on and call the `getTokens` method without waiting for the delay's completion event.

What I would expect to see is a use of the `rxDelay` on line 102 that's integrated into the main event stream. Was there a reason you didn't utilize the delay just above this `mergeMap`?

Reply:
I think I was a little confused about how the rxDelay worked. This has been corrected.